README.md

```diff
@@ -427,8 +428,8 @@ spec:
     rotationFrequency: 600h
  ```
 
-Once the `ServiceBinding` is rotated, the original binding will be updated with new credentials. 
-A new `ServiceBinding` named '<original-binding-name>-<guid>' will be created as a backup of the old credentials, which will be deleted according to the configured TTL. <br>
+Once the `ServiceBinding` is rotated, the original `ServiceBinding` will be updated with a new binding, the old binding will be saved in a new `ServiceBinding` 
+named '<original-binding-name>-<guid>', which will be deleted according to the configured TTL. <br>
 To see when the service binding was last rotated, refer to the `status.lastCredentialsRotationTime`. <br>
```

Comment:
for me it is quite confusing...
"original `ServiceBinding` will be updated with a new binding" - what does it mean to update binding with a new binding?
same for "the old binding will be saved in a new `ServiceBinding" - what does it really mean? the content of the old binding is copied to a new one?

Reply:
Changed